src/hotspot/cpu/x86/interp_masm_x86.hpp

```diff
@@ -212,11 +212,8 @@ class InterpreterMacroAssembler: public MacroAssembler {
   void verify_method_data_pointer();
 
   void set_mdp_data_at(Register mdp_in, int constant, Register value);
-  void increment_mdp_data_at(Address data, bool decrement = false);
-  void increment_mdp_data_at(Register mdp_in, int constant,
-                             bool decrement = false);
```

Comment:
`decrement` is never used?

Reply:
Yes, decrement is never used, it's a dead code.